serve/graph/resolver.go

```diff
@@ -0,0 +1,82 @@
+//go:generate go run github.com/99designs/gqlgen generate
+
+package graph
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/99designs/gqlgen/graphql"
+
+	"github.com/gnolang/tx-indexer/events"
+	"github.com/gnolang/tx-indexer/storage"
+	"github.com/gnolang/tx-indexer/types"
+)
+
+// This file will not be regenerated automatically.
+//
+// It serves as dependency injection for your app, add any dependencies you require here.
+
+const maxElementsPerQuery = 10000
+
+func dereferenceInt(i *int) int {
+	if i == nil {
+		return 0
+	}
+
+	return *i
+}
```

Comment:
```suggestion
func deref[T any](v *T) T {
	if v == nil {
		var zero T
		return zero
	}
	return *v
}
```

Playground: https://go.dev/play/p/7zF-CCVws84

To replace both this one and time?

Reply:
Maybe should go in schema.resolvers, as they're primarily used there.

Reply:
Good catch